sdk/spring/spring-integration-azure-servicebus/src/test/java/com/azure/spring/integration/servicebus/inbound/ServiceBusInboundChannelAdapterTests.java

```diff
@@ -24,8 +24,8 @@
 
 import static org.mockito.Mockito.mock;
 
-public class ServiceBusInboundAdapterTest {
-    //TODO(yiliu6): extends from InboundChannelAdapterTest
+public class ServiceBusInboundChannelAdapterTests {
```

Comment:
Do we still need this TODO?

Reply:
Maybe we can remove these two classes AbstractInboundChannelAdapter, InboundChannelAdapterTest?

Reply:
OK